study.md

```diff
@@ -50,7 +50,61 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+{
+  "cupboard"
+    "numberOfBowls": 8,
+    "numberOfPlates": 6
+}
+{
+  "electricStove"
+    "heatingElements": [
+      {
+        "extraLarge"
+      },
+      {
+        "large"
+      },
+      {
+        "medium"
+      },
+      {
+        "small"
+      }
+    ]
+  }
+  {
+    "spicesRack": [
+      {
+        "topShelf": [
```

Comment:
You can simply represent these items in an array like this: `["chives", "lemongrass"]` rather than as keys without a value